test-suite/squidconf/dstdomain-domain-dup.conf.instructions

```diff
@@ -0,0 +1 @@
+expect-message WARNING:.*is.duplicated.in.the.list.*You.should.remove.one
```

Comment:
Here and elsewhere, please include the problematic value into the expected message regex. We want to test not just the detection of a similar problem, but (to the extent possible) the detection of the exact/expected problem.